website/server/controllers/api-v3/chat.js

```diff
@@ -318,7 +318,7 @@ api.flagChat = {
     let message = _.find(group.chat, {id: req.params.chatId});
 
     if (!message) throw new NotFound(res.t('messageGroupChatNotFound'));
-
+    if (message.uuid === 'system') throw new BadRequest(res.t('messageCannotFlagSystemMessages', {communityManagerEmail: nconf.get('EMAILS:COMMUNITY_MANAGER_EMAIL')}));
```

Comment:
@pcvonz looks good! can you just move this nconf call to a const at the beginning of the file like here https://github.com/HabitRPG/habitica/pull/9408/files#diff-ade116c537945ae89e2f5f059d4d577fR21? thanks and sorry for the delay!

Reply:
Alright, should be good now!